packages/migration/src/migrations/v0037_removeTransientMediaProperties.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function removeTransientMediaProperties({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  if (!element.resource) {
+    return element;
+  }
+
+  delete element.resource.local;
```

Comment:
@spacedmonkey General question to understand this better:
In which case is a resource not `local` and not `isExternal` either? Just wondering that if it's one or another, why keep track of both and not just `isExternal` which could be `true` or `false`. 

Also, not for this PR but should it be renamed to `isLocal` for consistency?

Reply:
Local is being removed here. We can workout if a resource is local by checking if it is in the upload queue or not.